.gitignore

```diff
@@ -1,2 +1,2 @@
 build
-
+.vscode
```

Comment:
This is a user specific file that should be ignored in your global git ignore file.

Reply:
Right, done.
Still learning..